assets/js/phoenix_live_view.js

```diff
@@ -1003,8 +997,8 @@ export class View {
     if(!this.isLoading()){ this.showLoader(LOADER_TIMEOUT) }
     let linkRef = this.liveSocket.setPendingLink(href)
     this.pushWithReply("link", {url: href}, resp => {
-      if(resp.redirect){
-        this.onExternalLiveRedirect(href, linkRef)
+      if(resp.link_redirect){
```

Comment:
I renamed this to `link_redirect` so we can remove conditional check from `pushWithReply`.

Reply:
Maybe we should call it `hard_redirect`? We can issue it from the server, s `link` seems too specific?

Reply:
This value is returned exclusively for the "link" event, which is why I went with the `link_redirect` route.